src/wayland/protocol/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+set(GENERATED_FILES "")
+
+macro(GENERATE_PROTOCOL NAME_PREFIX PROTOCOL_NAME)
```

Comment:
Maybe this should move to `cmake/MirCommon.cmake`, and be used in the miral tests?

Reply:
Done